packages/block-library/src/navigation-link/edit.js

```diff
@@ -114,6 +113,41 @@ function NavigationLinkEdit( {
 		selection.addRange( range );
 	}
 
+	async function handleCreatePage( pageTitle ) {
+		const type = 'page';
+		const page = await saveEntityRecord( 'postType', type, {
+			title: pageTitle,
+			status: 'publish',
+		} );
+
+		// `page` may not reject the Promise
+		// but may still be invalid. Here we
+		// tests for unexpected values and throw accordingly.
+		if ( null === page || undefined === page ) {
+			throw new TypeError( 'API response returned invalid Page.', page );
+		}
+
+		const requiredEntityProps = [ 'id', 'title', 'link' ];
+
+		const pageMissingProperty = requiredEntityProps.find(
+			( pageProp ) => ! page.hasOwnProperty( pageProp )
+		);
+
+		if ( pageMissingProperty ) {
+			throw new TypeError(
+				`API response returned invalid page. Missing required property "${ pageMissingProperty }".`,
+				page
+			);
+		}
```

Comment:
It was noted in an earlier review: this should be removed.

Reply:
It was noted yes but we were discussing it and I don't recall ever arriving at a final decision.

I'm going to remove it now along with https://github.com/WordPress/gutenberg/pull/19775#discussion_r379588011.